tasks/main.yml

```diff
@@ -1,23 +1,12 @@
 ---
-- set_fact: rdbms=mariadb
-
+# default to mariadb, see vars/main.yml
 - set_fact: rdbms=mysql
   when: (ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS') and ansible_distribution_major_version <= 6
 
 - set_fact: rdbms_root_pw="{{ lookup('password', '~/' + rdbms + '_root_password length=15') }}"
   no_log: True
 
-- set_fact: rdbms_service=mariadb
-  when: rdbms == 'mariadb'
-
-- set_fact: rdbms_service=mysqld
-  when: rdbms == 'mysql'
-
-- set_fact: log_file=/var/log/mysql.log
-  when: rdbms == 'mysql'
-
-- set_fact: log_file=/var/log/mariadb/mariadb.log
-  when: rdbms == 'mariadb'
+- include_vars: "{{ rdbms }}.yml"
```

Comment:
as for this point, you can disagree and discard it as the maintainer. I just though it is different than the OS type because it is a "user" input and better have a "you entered a bad value" hint than doing something strange to debug.